modules/OCPP/OCPP.cpp

```diff
@@ -414,4 +410,28 @@ void OCPP::ready() {
     invoke_ready(*p_auth_provider);
 }
 
+ocpp1_6::ChargePointErrorCode OCPP::getOcppErrorCode(json evse_error) {
```

Comment:
here you can pass by const reference, otherwise the object will be copied (which doesn't really hurt here)
furthermore we have this name alias (using Object = nlohmann::json), it's a bit easier to deduct the intended type.
so here one should write
const Object& evse_error